eclass/libretro.eclass

```diff
@@ -49,7 +49,7 @@ if [[ ! ${PV} = 9999 ]] && [[ ! ${PN} = retroarch ]] && [[ ! ${PN} = ppsspp-libr
 fi
 
 # Workaround for ebuilds needing submodules
-if [[ ${PN} = ppsspp-libretro ]] || [[ ${PN} = psp1-libretro ]]  || [[ ${PN} = citra-libretro ]] && [[ ! ${PV} = 9999 ]]; then
```

Comment:
This caused us to include git-r3, which requires empty keywords.
And I believe it is no longer needed, since everything is included in the source tarball.